packages/api/src/functions/dbAuth/DbAuthHandler.ts

```diff
@@ -65,10 +65,18 @@ interface SessionRecord {
 
 type AuthMethodNames = 'login' | 'logout' | 'signup' | 'getToken'
 
+type Params = {
+  username: string | undefined
```

Comment:
Don't have full context but just wanted to point out subtle differences here:

```
username?: string
password?: string
``` 

would mean we could call this like `new DbAuthHandler(event, context, { method: 'xxx'})` and this would be valid

But `string | undefined` means that you must pass an object that contains the key username and password, even if they're undefined.